config/crd/bases/infinispan.org_infinispans.yaml

```diff
@@ -886,8 +886,8 @@ spec:
                 - minReplicas
                 type: object
               cloudEvents:
-                description: InfinispanCloudEvents describes how Infinispan is connected
-                  with Cloud Event, see Kafka docs for more info
+                description: Deprecated. Has no effect starting with Infinispan 15.0.0
```

Comment:
@ryanemerson this line changed when I compiled the code. I'm not sure if I should revert it or not.

Reply:
I've pushed the update to `main` :+1: